lib/events/Events.ts

```diff
@@ -14,30 +18,50 @@ export interface Events {
  * Right now doesn't use GraphQL
  *
  */
-export class GraphQLEventItems implements Events {
+export class GraphQLEvents implements Events {
   eventsWithIds (ids: string[]): Event[] {
     if (ids.length === 0) return []
     const eventsList = []
     const date = new Date()
-    date.setHours(date.getHours() + 10)
+    const date2 = new Date()
+    const address: Placemark = {
+      name: "UCSC Campus",
+      country: "United States of America",
+      postalCode: "95064",
+      street: "High St",
+      streetNumber: "1156",
+      region: "CA",
+      isoCountryCode: "US",
+      city: "Santa Cruz"
+    }
+    date.setHours(10, 30)
+    date.setDate(date.getDate() + 0)
+    date2.setHours(16, 30)
+    date2.setDate(date2.getDate() + 2)
 
     for (let i = 0; i < ids.length; i++) {
       const event: Event = {
-        id: String(ids[i]),
+        id: i,
         userId: "3234324",
-        username: "Test Event",
-        title: "Title for Event",
+        username: "Nicolette Antisdel",
+        title: "Pickup Basketball",
         repliesCount: 2,
         writtenByYou: true,
-        startTime: date,
-        maxOccupancy: 5,
-        isAcceptingInvitations: true,
-        colorHex: "magenta",
-        distance: 0.5
+        duration: new FixedDateRange(date, date2),
```

Comment:
Nit: There is a helper function called `dateRange` which exists to make a `FixedDateRange` in less keystrokes instead of doing `new FixedDateRange(...)`.

Reply:
Yeah, I used dateRange in Event and must've forgot to change the one in Events.